packages/trader/src/AppV2/Components/TradeParameters/RiskManagement/take-profit-and-stop-loss-input.tsx

```diff
@@ -51,10 +53,21 @@ const TakeProfitAndStopLossInput = ({
         trade_types,
         trade_type_tab,
         onChangeMultiple,
+        proposal_info,
         validation_params,
+        validation_errors,
     } = trade_store;
 
     const is_take_profit_input = type === 'take_profit';
+    const contract_types = getDisplayedContractTypes(trade_types, contract_type, trade_type_tab);
+
+    // For tracking errors, that are coming from proposal for take profit and stop loss
+    const { message } = useTradeParamError({
```

Comment:
For scenarios, when we haven't change the TP or SL, but already have an error for their fields (e.g. in case if Stake was changed).